doc/classes/RenderingServer.xml

```diff
@@ -2630,6 +2630,15 @@
 				  - Position + Custom data: 16 floats (12 floats for Transform3D, 4 floats of custom data)
 				  - Position + Vertex color + Custom data: 20 floats (12 floats for Transform3D, 4 floats for Color, 4 floats of custom data)
 				[/codeblock]
+				All the transforms i.e Transform2D and Transform3D should be in Row-Major form.
+				Example:
+				If the Transform3D is:
+				[codeblock lang=text]
+				1 0 0 x
+				0 1 0 y
+				0 0 1 z
+				[/codeblock]
+				then the [PackedFloat32Array] will be (1, 0, 0, x, 0, 1, 0, y, 0, 0, 1, z)
```

Comment:
The identity basis in the example might be a little unclear (e.g. `(1, 0, 0)` part matches both first row and first column; row-major order is mentioned above but still might be better to be explicit).

Also Transform2D consists of 6 floats so it should be said where the 2 additional padding floats go.

I suggest full listings, using properties:
```suggestion
				Instance transforms are in row-major order. Specifically:
				- For [Transform2D] the float-order is: [code](x.x, y.x, padding_float, origin.x, x.y, y.y, padding_float, origin.y)[/code].
				- For [Transform3D] the float-order is: [code](basis.x.x, basis.y.x, basis.z.x, origin.x, basis.x.y, basis.y.y, basis.z.y, origin.y, basis.x.z, basis.y.z, basis.z.z, origin.z)[/code].
```

I'm not sure if the padding floats in Transform2D data can be any value or if maybe they need to be `0.0`. :thinking:

Reply:
yes, thats better for sure.